ambry-api/src/main/java/com.github.ambry/account/AccountUpdateListener.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright 2017 LinkedIn Corp. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ */
+package com.github.ambry.account;
+
+import java.util.Collection;
+
+
+/**
+ * A {@code Listener} can subscribe for {@link Account} creation and update through {@link AccountService}.
+ */
+public interface AccountUpdateListener {
```

Comment:
Took a look and also had a discussion with @cgtz . Concluded that if we use `Consumer` as the listener, the current way would be the simplest.